src/compose/ComposeBox.js

```diff
@@ -122,6 +125,11 @@ class ComposeBox extends PureComponent<Props, State> {
     selection: { start: 0, end: 0 },
   };
 
+  componentWillUnmount() {
+    clearTimeout(this.messageBlurTimeoutId);
+    clearTimeout(this.topicBlurTimeoutId);
```

Comment:
I'm surprised this works! Apparently `clearTimeout({anything: "goes"})` is accepted without comment. (In the browser, at least.)